src/atoms/forms/select.test.tsx

```diff
@@ -1,5 +1,72 @@
 import React from "react";
+import { Select } from "./select";
+import faker from "faker";
+import { render, fireEvent } from "@testing-library/react";
 
 describe("Select", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/11", () => {});
+    test("when default props, renders with options", () => {
+        // Arrange
+        const expected = faker.random.word();
+        const testLabel = faker.random.word();
+        const testValue = faker.random.word();
+
+        // Act
+        const { getByText } = render(
+            <Select
+                id={expected}
+                options={[{ label: testLabel, value: testValue }]}
```

Comment:
Thought - not sure if we are using this `SelectOption` interface anywhere else, but if so, it might be worth breaking out a factory for so we can clean up some of this arrangement

Reply:
After further looking at the files, I don't think it is used often enough to break it out, but I still can if you would like. 👍